bvm/ballerina-profiler/spotbugs-exclude.xml

```diff
@@ -20,4 +20,8 @@
         <Class name="io.ballerina.runtime.profiler.Profiler"/>
         <Bug pattern="DP_CREATE_CLASSLOADER_INSIDE_DO_PRIVILEGED"/>
     </Match>
+    <Match>
+        <Class name="io.ballerina.runtime.profiler.ui.JsonParser$Data"/>
+        <Bug pattern="URF_UNREAD_FIELD"/>
```

Comment:
Can't we remove the unread field from the class?

Reply:
This is intenral used in gson. Not used in profile code.